src/routes/groups/groups.router.js

```diff
@@ -0,0 +1,87 @@
+const express = require('express');
+const { listGroup, postGroup, updateGroup, deleteGroup } = require("./groups.controller");
+const { check, param } = require('express-validator');
+const   { validateBody } = require('../../middleware/validate-body');
+
+const router = express.Router();
+/**
+ * @swagger
+ * /groups:
+ *   get:
+ *     description: Retreive a list of community groups
+ *     produces:
+ *      - application/json
+ *     responses:
+ *       200:
+ *         description: List of community groups successfully retrieved
+ *         schema:
+ *           type: array
+ */
+router.get("", listGroup);
```

Comment:
Well-designed RESTful APIs usually have at least one path component in a resource url to indicate what type of resource lives at that url.  Since you only have one type of resource now, using an empty path works, but as soon as you have more than one, you're going to want to put a label in the path. What do you think would be a good path for the Group resource?